gapic/api/artman_lro.yaml

```diff
@@ -0,0 +1,25 @@
+common:
+  api_name: google-longrunning-operations
```

Comment:
I think I would prefer "google-longrunning". It matches the directory it's declared in.